x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_suggestions.test.ts

```diff
@@ -380,6 +380,45 @@ describe('xy_suggestions', () => {
     });
   });
 
+  test('overwrites column to dimension mappings if a date dimension is added', () => {
+    (generateId as jest.Mock).mockReturnValueOnce('dummyCol');
+    const currentState: XYState = {
+      isHorizontal: false,
+      legend: { isVisible: true, position: 'bottom' },
+      preferredSeriesType: 'bar',
+      layers: [
+        {
+          accessors: ['price', 'quantity'],
+          layerId: 'first',
+          seriesType: 'bar',
+          splitAccessor: 'dummyCol',
+          xAccessor: 'product',
+        },
+      ],
+    };
+    const [suggestion, ...rest] = getSuggestions({
```

Comment:
I think some of the brittleness in our suggestion tests is due to asserting based on the position of suggestions in the response object. Instead, I think we can use `contains` like Chris did here: https://github.com/elastic/kibana/pull/46625